nemo/collections/llm/api.py

```diff
@@ -95,6 +105,13 @@ def train(
         >>> llm.train(model, data, trainer, tokenizer="data")
         PosixPath('/path/to/log_dir')
     """
+    # [ModelOpt]: If modelopt_state exists, overwrite transformer_layer_spec to modelopt spec
+    if resume:
+        if resume.restore_config and resume.restore_config.path:
+            set_modelopt_spec_if_exists_in_ckpt(model, resume.restore_config.path)
+        elif resume.resume_from_path:
+            set_modelopt_spec_if_exists_in_ckpt(model, resume.resume_from_path)
```

Comment:
@yashaswikarnati is there any utility function we can use to get path based on `resume.restore_config.path`, `resume.resume_from_path` or `resume.resume_from_dir`?